packages/stops-overlay/src/index.js

```diff
@@ -9,13 +9,14 @@ import { FeatureGroup, MapLayer, withLeaflet } from "react-leaflet";
 class StopsOverlay extends MapLayer {
   componentDidMount() {
     // set up pan/zoom listener
-    this.props.leaflet.map.on("moveend", () => {
-      this.refreshStops();
-    });
+    this.props.leaflet.map.on("moveend", this.refreshStops);
   }
 
   // TODO: determine why the default MapLayer componentWillUnmount() method throws an error
-  componentWillUnmount() {}
+  componentWillUnmount() {
+    // Remove the pan/zoom listener set up above only.
```

Comment:
Not sure I understand the last part of this comment "... set up above only."

Reply:
Updated.